### PR TITLE
Hotfix/nuget packages

### LINKED ISSUE #16862: TestPathCommand and several others removed in PowerShell 7.2 SDK
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Using C# SDK, I updated `Microsoft.PowerShell.Commands.Management` from `7.1.5` to `7.2.1`.

The SDK no longer has references to the following commands:

1. JoinPathCommand
2. TestPathCommand
3. GetChildItemCommand
4. GetItemCommand
5. SetItemCommand

If this was intentional, how should I be referencing these commands?

### Expected behavior

```console
I need to import the commands listed under steps to reproduce
```

### Actual behavior

```console
I can now longer use the commands listed under steps to reproduce
```

### Error details

_No response_

### Environment data

```powershell
Key                       Value
---                       -----
PSVersion                 7.2.1
PSEdition                 Core
GitCommitId               7.2.1
OS                        Linux 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021
Platform                  Unix
PSCompatibleVersions      {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion 2.3
SerializationVersion      1.1.0.1
WSManStackVersion         3.0
```

### Visuals

_No response_

### PR DESCRIPTION
**Pull request by dkattan**
# PR Summary

Should fix https://github.com/PowerShell/PowerShell/issues/16862

Microsoft.PowerShell.SDK is missing references to
* Microsoft.PowerShell.Commands.Management
* Microsoft.PowerShell.Commands.Diagnostics
* Microsoft.PowerShell.Security
* Microsoft.WSMan.Management

This was OK because you could manually specify the a Reference via a HintPath but this workaround broke in 7.2 preview 7 due to this change: https://github.com/PowerShell/PowerShell/commit/046dec4d7c5eba2b00e7ee8c1fe5efeb9e620898#r98046816

Since then, it is literally impossible to reference types in any of those 4 libraries like ResolvePathCommand among dozens of others.

The only way I was able to work around this was by using a hex editor to modify the Microsoft.PowerShell.Commands.Management.DLL file to reference System.Management.Automation 7.X.X.0 instead of 7.X.X.500

Additionally, added symbols to the nuget packages so SourceLink could work which will help app developers using the PowerShell SDK tremendously.

This change should not have any negative consequences as the nuget packages are unusable anyway, however I don't have a great way to test this as the devops pipeline only works in the Microsoft DevOps environment.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by dkattan** [comment]
@TravisEz13 Can you take a look at this and let me know what you think?

**Comment by TravisEz13** [comment]
@adityapatwardhan  Can you review?

File: test/perf/benchmarks/assets/compiler.test.ps1 [hunk 1]
```diff
@@ -2278,6 +2278,7 @@ function Start-PSPackage {
                     PackageVersion = $Version
                     PackageRuntime = $Runtime
                     PackageConfiguration = $Configuration
+                    IncludeSymbols = $IncludeSymbols
```
Comment (adityapatwardhan) [inline test/perf/benchmarks/assets/compiler.test.ps1:2281]: Revert this. This is just a test file.

**Comment by adityapatwardhan** [comment]
I have some concerns about the size of the nuget packages with symbols added in. 

I am starting a private build for the change to see if it fixes it. My hunch is that it would need some code to be added here:

https://github.com/PowerShell/PowerShell/blob/c87f2a48fd8ba2a3a1c54d9a82d6937dc57f44c2/tools/packaging/packaging.psm1#L2542-L2624

File: test/perf/benchmarks/assets/compiler.test.ps1 (hunk 1 above)
Comment (dkattan) [reply test/perf/benchmarks/assets/compiler.test.ps1:2281]: But shouldn't we also test this?

Comment (adityapatwardhan) [reply test/perf/benchmarks/assets/compiler.test.ps1:2281]: This is used only for testing the compiler. Please remove this.

**Comment by daxian-dbw** [comment]
Closing this PR as @adityapatwardhan is actively working on a solution to this issue.

**Pull request closed without merging**